src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -496,6 +502,11 @@ private String postRequest(PostMethod httppost) throws UnsupportedEncodingExcept
 
     }
 
+    private <T> String serialize(T o) throws IOException {
```

Comment:
Oh, sorry @kmadel. I was thinking about parsing objects but you are serializing here. You can omit this comment.